instrumentation/weblogic/src/main/java/com/splunk/opentelemetry/instrumentation/weblogic/metrics/ServerLifecycleListenerInstrumentation.java

```diff
@@ -39,7 +39,7 @@ public void transform(TypeTransformer transformer) {
 
   @SuppressWarnings("unused")
   public static class InitializeAdvice {
-    @Advice.OnMethodExit(suppress = Throwable.class)
+    @Advice.OnMethodEnter(suppress = Throwable.class)
```

Comment:
Was this change to also get the process going earlier in the lifecycle?

Reply:
Yes. The startup message should be printed by something called from this method.